samples/dnn/object_tracker.cpp

```diff
@@ -0,0 +1,355 @@
+            params.backbone = findModel(backbone, "");
+            params.neckhead = findModel(headneck, "");
+            params.backend = backend;
+            params.target = target;
+            tracker = TrackerNano::create(params);
+        }
+        catch (const cv::Exception& ee)
+        {
+            std::cerr << "Exception: " << ee.what() << std::endl;
+            std::cout << "Can't load the network by using the following files:" << std::endl;
+            std::cout << "backbone : " << backbone << std::endl;
+            std::cout << "headneck : " << headneck << std::endl;
+            return 2;
+        }
+        trackObject("nanoTracker", tracker, inputName);
+        trackObject("DaSiamRPNTracker", tracker, inputName);
+    }
+    else if (modelName == "dasiamrpn" || parser.get<String>("dasiamrpn_model") != ""){
+        net = parser.get<String>("dasiamrpn_model");
+        kernel_cls1 = parser.get<String>("kernel_cls1");
+        kernel_r1 = parser.get<String>("kernel_r1");
+
+        Ptr<TrackerDaSiamRPN> tracker;
+        try
+        {
+            if (net == "" || kernel_cls1 == "" || kernel_r1 == ""){
+                cout<<"Pass model files using --dasiamrpn_model , --kernel_cls1 and --kernel_r1 arguments for using dasiamrpn tracker. \nDownload dasiamrpn_model using link: https://www.dropbox.com/s/rr1lk9355vzolqv/dasiamrpn_model.onnx?dl=0"<<endl;
+                cout<<"And, download kernel_r1 using link: https://www.dropbox.com/s/999cqx5zrfi7w4p/dasiamrpn_kernel_r1.onnx?dl=0"<<endl;
+                cout<<"And, download kernel_cls1 using link: https://www.dropbox.com/s/qvmtszx5h339a0w/dasiamrpn_kernel_cls1.onnx?dl=0"<<endl;
+                return -1;
+            }
+            cout<<"Using Dasiamrpn Tracker."<<endl;
+            TrackerDaSiamRPN::Params params;
+            params.model = findModel(net, "");
+            params.kernel_cls1 = findModel(kernel_cls1, "");
+            params.kernel_r1 = findModel(kernel_r1, "");
+            params.backend = backend;
+            params.target = target;
+            tracker = TrackerDaSiamRPN::create(params);
+        }
+        catch (const cv::Exception& ee)
+        {
+            cerr << "Exception: " << ee.what() << endl;
+            cout << "Can't load the network by using the following files:" << endl;
+            cout << "siamRPN : " << net << endl;
+            cout << "siamKernelCL1 : " << kernel_cls1 << endl;
+            cout << "siamKernelR1 : " << kernel_r1 << endl;
+            return 2;
+        }
+        trackObject("DaSiamRPNTracker", tracker, inputName);
+    }
+    else{
+        cout<<"Pass the valid alias. Choices are { nano, vit, dasiamrpn }"<<endl;
+        return -1;
+    }
+    return 0;
+}
+
+
+int main(int argc, char **argv)
+{
+    try
+    {
+        return run(argc, argv);
+    }
+    catch (const exception& e)
+    {
+        cerr << "FATAL: C++ exception: " << e.what() << endl;
+        return 1;
+    }
+}
```

Comment:
Such main does not make a lot of sense. I propose to rename "run" to "main" and remove it.